pageserver/client/src/mgmt_api.rs

```diff
@@ -92,11 +92,11 @@ impl Client {
     ///
     /// Use/add one of the properly typed methods below if you know aren't proxying, and
     /// know what kind of response you expect.
-    pub async fn get_raw(&self, path: String) -> Result<reqwest::Response> {
+    pub async fn op_raw(&self, method: Method, path: String) -> Result<reqwest::Response> {
```

Comment:
you should also change the rustdoc comment above

Reply:
I meant the "Get an arbitrary path" part of the comment :). The part you deleted was fine :)

Reply:
fixed in https://github.com/neondatabase/neon/pull/11401/commits/bb594c4747200b94af40518bd9b18809cb5fdd42 :)